### ISSUE TITLE
is proton 7.0-4  supported?

### ISSUE THREAD

**Issue by GerardoBaeza** [bcook254/apex-legends-cache#80]
With the latest  release of proton 7 updated dxvk version to 1.10.3. is this enough to make it work with the current cache? (asking because i have a lot of issues with mangohud and proton experimental)

proton 7.0-4 patch notes https://github.com/ValveSoftware/Proton/releases/tag/proton-7.0-4

**Comment by bcook254** [comment]
I would assume so yes. I have not had a chance to personally test it yet
but since it comes right from the experimental branch it should be fine.

**Comment by serossi** [comment]
no its not, on 7.0.4 the file gets overwritten on each start of the game. dxvk version change you need experimental.

only option on 7.0.4 is to build it yourself, i recommend using a smurf for this as you will die a lot until its playable. also its easier in newbie lobbys to explore :))

**Comment by deep2p4ce** [comment]
Yes you are right. Proton 7.0-4 isn't "compatible" with this repos' cache files. I tested it while trying to figure out why my games started to stutter after ~30mins after a system update (after a lot of downgrading packages and searching the web, I found out it was caused by MangoHud...). And as you said, starting Apex with Proton 7.0-4 deletes the local cache file and creates a new one. But this cache is of version V10 and thus not compatible with the cache of this repo which is V15. But Proton-Experimental works.
